kairon/shared/data/processor.py

```
@@ -8957,6 +8958,73 @@
+
+        content_dir = os.path.join('file_content_upload_records', bot)
+        Utility.make_dirs(content_dir)
+        file_path = os.path.join(content_dir, file_content.filename)
+
+        with open(file_path, "wb") as buffer:
+            shutil.copyfileobj(file_content.file, buffer)
+
+        file_content.file.seek(0)
+
+        if type == UploadHandlerClass.crud_data:
+            if file_content.content_type not in valid_csv_types and not file_content.filename.lower().endswith('.csv'):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)